apps/frontend/src/libs/components/button/button.tsx

```diff
@@ -1,11 +1,48 @@
+import { AppRoute } from "~/libs/enums/enums.js";
+import { type ValueOf } from "~/libs/types/types.js";
+
+import { Link } from "../components.js";
+import styles from "./styles.module.css";
+
 type Properties = {
+	color?: "primary";
+	href?: ValueOf<typeof AppRoute>;
+	isFluid?: boolean;
```

Comment:
```suggestion
	style?: 'fullfiled' | 'outlined';
```
default is `fullfiled`

Reply:
I added filled and outlined styles. But isFluid flag was added for making button width 100% (by default width is auto). How can I set 100% width for button correctly? (it's not for my task, it's for others)
Or I can skip it on this task?